taskt/Core/Automation/Commands/Data Commands/LogDataCommand.cs

```diff
@@ -91,8 +87,7 @@ public override string GetDisplayValue()
                 logFileName = "taskt " + v_LogFile + " Logs.txt";
             }
 
-
-            return base.GetDisplayValue() + " [Write Log to 'taskt\\Logs\\" + logFileName + "']";
+            return base.GetDisplayValue() + $" [Write Log '{v_LogText}' to 'taskt\\Logs\\{logFileName}']";
```

Comment:
When I tested this with a file called "test", the display value produced 'taskt\Logs\taskt test Logs.txt'. I think something went wrong here.

Reply:
Changed logFileName formatting (in if-else part)